Content.Shared/_DV/Inventory/InventorySystem.Relay.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Shared._DV.Chemistry.Systems;
+
+namespace Content.Shared.Inventory;
+
+public partial class InventorySystem
+{
+    public void InitializeRelayDV()
+    {
+        // by-ref events
+        SubscribeLocalEvent<InventoryComponent, SafeSolutionThrowEvent>(RefRelayInventoryEvent);
```

Comment:
move this to the other system and delete this file
use _inventory.RefRelayInventoryEvent